File: atcoder/abc179B_go_to_jail.py
# unihernandez22
# https://atcoder.jp/contests/abc179/tasks/abc179_b
# implementation

n = int(input())

count = 0
for _ in range(n):
    a, b = input().split()
    if a == b:
        count += 1
    else:
        count = 0
    if count == 3:
        print("Yes")
        quit()

print("No")

File: atcoder/abc179B_go_to_jail.cpp
// https://atcoder.jp/contests/abc179/tasks/abc179_b
#include<iostream>

using namespace std;

int main() {
    int n, a, b;
    cin >> n;
    int count = 0;
    while (n > 0) {
        cin >> a >> b;
        if (a == b) count += 1;
        else count = 0;

        if (count == 3) {
            cout << "Yes" << endl;
            break;
        }
        n--;
    }

    if (count < 3) cout << "No" << endl;

    return 0;
}
